src/ORM/DataObject.php

```diff
@@ -3082,8 +3082,11 @@ public static function get(
      *
      * @return DataObject|null The first item matching the query
      */
-    public static function get_one($callerClass, $filter = "", $cache = true, $orderby = "")
+    public static function get_one($callerClass = null, $filter = "", $cache = true, $orderby = "")
```

Comment:
LGTM, the docblock for the callerClass argument could do with a little tweak tho

Reply:
Updato